tests/data/l8biome/data.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import shutil
+from typing import Dict, List, Union
+
+import numpy as np
+import rasterio
+from rasterio import Affine
+from rasterio.crs import CRS
+
+SIZE = 36
+
+np.random.seed(0)
+
+FILENAME_HIERARCHY = Union[Dict[str, "FILENAME_HIERARCHY"], List[str]]
+
+bands = [
+    "B1.TIF",
+    "B2.TIF",
+    "B3.TIF",
+    "B4.TIF",
+    "B5.TIF",
+    "B6.TIF",
+    "B7.TIF",
+    "B8.TIF",
+    "B8A.TIF",
+    "B9.TIF",
+    "B10.TIF",
+    "B11.TIF",
+    "B12.TIF",
+]
+
+barren_prefix = "LC80420082013220LGN00"
+forest_prefix = "LC80070662014234LGN00"
+
+filenames: FILENAME_HIERARCHY = {
+    "barren": {
+        barren_prefix: []
+    },
+    "forest": {
+        forest_prefix: []
+    }
+}
+
+for band in bands:
+    filenames["barren"][barren_prefix].append(f'{barren_prefix}_{band}')
+    filenames["forest"][forest_prefix].append(f'{forest_prefix}_{band}')
+
+ROOT = f"../../../../BC/{barren_prefix}"
+FILENAME = f"{barren_prefix}_B1.TIF"
+src = rasterio.open(os.path.join(ROOT, FILENAME))
```

Comment:
If I'm understanding correctly, this relies on real data existing on the system. We should instead hardcode the driver/dtype/src/transform in `data.py`. Someday, I'm planning on moving all of this fake data generation into the dataset fixtures so we don't have to store any fake files in the repo.